gimme

```diff
@@ -100,15 +100,18 @@ _extract() {
 _compile() {
 	(
 		local bootstrap_root=""
-		for bootstrap_root in \
+		for candidate in \
+			"${GIMME_VERSION_PREFIX}/go1.4.1.${GIMME_OS}.${GIMME_ARCH}" \
+			"${GIMME_VERSION_PREFIX}/go1.4.1.${GIMME_OS}.${GIMME_HOSTARCH}" \
 			"${GIMME_VERSION_PREFIX}/go1.4.${GIMME_OS}.${GIMME_ARCH}" \
 			"${GIMME_VERSION_PREFIX}/go1.4.${GIMME_OS}.${GIMME_HOSTARCH}" ; do
-			if [ -d "${bootstrap_root}" ] ; then
+			if [ -d "${candidate}" ] ; then
+				bootstrap_root="${candidate}"
```

Comment:
This seems fine, but an alternative idea (that'll catch future possible Go 1.4 versions too) would be:
```bash
local bootstrap_root=""
local bootstrap_candidates=( "${GIMME_VERSION_PREFIX}/go1.4"*".${GIMME_OS}.${GIMME_ARCH}" )
if [ "${#bootstrap_candidates[@]}" -gt 0 ]; then
    bootstrap_root="$bootstrap_candidates" # take the first item off the array!
fi
```

(which would avoid the loop entirely also)